sale_order_merge/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+Merge sale orders
+=================
+
+This module allows the sale employee to merge draft or confirmed orders
+from the same customer.
+
+When orders are merged, draft invoices and unprocessed outgoing pickings
```

Comment:
Yeah, you're right. You can't merge pickings without merging the "upstream" sales orders, or the links will not be totally correct (although the main control is through sales order lines, not through sales orders, but the main information/reports won't be correct).